public/client.js

```diff
@@ -117,13 +117,15 @@ socket.on('Display Message', function(data) {
     }
     // Adding Emoji
     var p = data.msg;
+    var p_notif=p;
     var colon1 = p.indexOf(":");
     while (colon1 != -1) {
         var colon2 = p.indexOf(":", colon1 + 1);
         if (colon2 != -1) {
             emoji_name = p.slice(colon1 + 1, colon2);
             position = emoji_names.indexOf(emoji_name)
             if (position != -1) {
+                p_notif = p_notif.slice(0, colon1+1) + emoji_name +p.slice(colon2);
```

Comment:
This line is not required, it's redundant. You can remove it.

Reply:
Yeah @yatharthMogra because data.msg(=p,=p_notif) has message in form of `text and :emoji:`. So you can remove it.